packages/plugin-client-drizzle/test/integration.ava.ts

```diff
@@ -0,0 +1,2187 @@
+});
+
+const orders = pgTable('orders', {
+  id: serial('id').primaryKey(),
+  region: text('region').notNull(),
+  product: text('product').notNull(),
+  amount: integer('amount').notNull(),
+  quantity: integer('quantity').notNull()
+});
+
+const network = pgTable('network_table', {
+  inet: inet('inet').notNull(),
+  cidr: cidr('cidr').notNull(),
+  macaddr: macaddr('macaddr').notNull(),
+  macaddr8: macaddr8('macaddr8').notNull()
+});
+
+const salEmp = pgTable('sal_emp', {
+  name: text('name'),
+  payByQuarter: integer('pay_by_quarter').array(),
+  schedule: text('schedule').array().array()
+});
+
+const _tictactoe = pgTable('tictactoe', {
+  squares: integer('squares').array(3).array(3)
+});
+
+const usersMigratorTable = pgTable('users12', {
+  id: serial('id').primaryKey(),
+  name: text('name').notNull(),
+  email: text('email').notNull()
+});
+
+interface Context {
+  id: string;
+  db: XataDatabase;
+  client: XataClient;
+  api: XataApiClient;
+}
+
+const test = anyTest as TestFn<Context>;
+
+test.before(async (t) => {
+  const ctx = t.context;
+  ctx.api = new XataApiClient({ apiKey, fetch, host, clientName: 'sdk-tests' });
+});
+
+test.beforeEach(async (t) => {
+  const ctx = t.context;
+  ctx.id = Date.now().toString(36);
+
+  const { databaseName: database } = await ctx.api.database.createDatabase({
+    workspace,
+    database: `sdk-integration-test-drizzle-${ctx.id}`,
+    data: { region },
+    headers: { 'X-Xata-Files': 'true' }
+  });
+
+  const { edits } = await ctx.api.migrations.compareBranchWithUserSchema({
+    workspace,
+    region,
+    database,
+    branch: 'main',
+    schema: { tables: tables as any }
+  });
+
+  await ctx.api.migrations.applyBranchSchemaEdit({ workspace, region, database, branch: 'main', edits });
+
+  const workspaceUrl = getHostUrl(host, 'workspaces').replace('{workspaceId}', workspace).replace('{region}', region);
+
+  ctx.client = new XataClient({
+    databaseURL: `${workspaceUrl}/db/${database}`,
+    branch: 'main',
+    apiKey,
+    fetch,
+    clientName: 'sdk-tests'
+  });
+
+  ctx.db = drizzle(ctx.client, { logger: ENABLE_LOGGING });
+});
+
+test.afterEach.always(async (t) => {
+  const ctx = t.context;
+
+  await ctx.api.database.deleteDatabase({ workspace, database: `sdk-integration-test-drizzle-${ctx.id}` });
+});
+
+test.skip('select all fields', async (t) => {
+  const { db } = t.context;
+
+  const now = Date.now();
+
+  await db.insert(usersTable).values({ name: 'John' });
+  const result = await db.select().from(usersTable);
+
+  t.assert(result[0]!.createdAt instanceof Date);
+  t.assert(Math.abs(result[0]!.createdAt.getTime() - now) < 100);
+  t.deepEqual(result, [{ id: 1, name: 'John', verified: false, jsonb: null, createdAt: result[0]!.createdAt }]);
+});
+
+test.skip('select sql', async (t) => {
+  const { db } = t.context;
+
+  await db.insert(usersTable).values({ name: 'John' });
+  const users = await db
+    .select({
+      name: sql`upper(${usersTable.name})`
```

Comment:
We return what PostgreSQL returns. The column name is `upper` and the value is `JOHN`. The issue is in the adapter.